examples/cfd/04_burgers.ipynb

```diff
@@ -39,20 +39,25 @@
     "%matplotlib inline\n",
     "\n",
     "# Some variable declarations\n",
-    "nx = 41\n",
-    "ny = 41\n",
-    "nt = 120\n",
+    "nx = 61  # Grid size on x axis\n",
+    "ny = 61  # Grid size on y axis\n",
+    "nt = 640 # Number of timesteps\n",
```

Comment:
So propagate 25 times longer right?

Reply:
what do you mean 5.6. nt is 5 times bigger and 5 batches of it so that's 25 times more

Reply:
It is 5* ( 640/120 ) = 5 *  5.333 = 26.666 . Both of us confused I guess..